first commit

ValeriyaGH opened the pull request.

Commit:
first commit

egorvas commented on pom.xml:
```diff
@@ -0,0 +1,43 @@
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>junit</groupId>
```
**Нужно исправить:** не нужно дублировать зависимости, всегда можно подобрать необходимый scope

ValeriyaGH replied:
убрала дублирующую зависимость

egorvas commented on src/main/java/ru/praktikum/scooter/pom/MainPage.java:
```diff
@@ -0,0 +1,44 @@
+        webDriver.findElement(acceptCookieButton).click();
+
+}
+public void clickToMakeOrderButton(int button) {
```
**Можно лучше:** в таких местах лучше использовать не int 1/2, а ENUM

egorvas commented on src/test/java/ru/praktikum/scooter/TestMakingOrder.java:
```diff
@@ -0,0 +1,83 @@
+@After
+public void quit(){
+        driver.quit();
+}
```
**Можно лучше:** старайся использовать корректную табуляцию

ValeriyaGH replied:
поправила

egorvas commented on src/test/java/ru/praktikum/scooter/TestOfQuestions.java:
```diff
@@ -0,0 +1,108 @@
+    @Parameterized.Parameters
+    public static Object[][] getQuestionsAndAnswers(){
+        return new Object[][]{
+                {"accordion__heading-0", false, "Сутки — 400 рублей. Оплата курьеру — наличными или картой."},
```
**Нужно исправить:** по условиям задания все локаторы должны быть вынесены в PO

ValeriyaGH replied:
оставила в параметрах только цифры и ответы

egorvas commented on src/test/java/ru/praktikum/scooter/TestOfQuestions.java:
```diff
@@ -0,0 +1,108 @@
+    //соответствие текста ответа ожидаемому
+    private final String accordeonTextExpected;
+
+    public TestOfQuestions(String accordeonId, boolean accordeonPanelIsVisible, String accordeonTextExpected){
```
**Можно лучше:** если в параметр передается всегда одно и то же значение, то зачем нужен этот параметр?
Я про accordeonPanelIsVisible

ValeriyaGH replied:
поправила

egorvas commented on src/test/java/ru/praktikum/scooter/TestOfQuestions.java:
```diff
@@ -0,0 +1,108 @@
+
+
+    @Test
+    public void QuestionsTest(){
```
**Нужно исправить:** функции java всегда с маленькой буквы

ValeriyaGH replied:
поправила

egorvas commented on src/test/java/ru/praktikum/scooter/TestOfQuestions.java:
```diff
@@ -0,0 +1,108 @@
+
+
+
+
```
**Можно лучше:** старайся удалять лишние строки

ValeriyaGH replied:
поправила

Commit:
внесены правки:
оптимизированы импорты и зависимости
оптимизированы параметры теста TestOfQuestions

Commit:
убрала лишний класс
